IPython/html/nbextensions.py

```diff
@@ -238,25 +289,36 @@ class NBExtensionApp(BaseIPythonApplication):
     
     overwrite = Bool(False, config=True, help="Force overwrite of existing files")
     symlink = Bool(False, config=True, help="Create symlinks instead of copying files")
+    user = Bool(False, config=True, help="Whether to do a user install")
+    prefix = Unicode('', config=True, help="Installation prefix")
+    nbextensions = Unicode('', config=True, help="Full path to nbextensions (probably use prefix or user)")
     verbose = Enum((0,1,2), default_value=1, config=True,
         help="Verbosity level"
     )
     
+    def check_install():
+        if sum(map(bool, [user, prefix, nbextensions])) > 1:
+            raise TraitError("Cannot specify more than one of user, prefix, or nbextensions.")
```

Comment:
self is missing, yes. This is duplicated because the ValueError is treated as a crash, where TraitError isn't. I think I can make it better without duplicating the check.